python-sdk/tests/e2e/conftest.py

```
@@ -0,0 +1,135 @@
+                "%s: prompt already gone (404), skipping",
+                context,
+                extra={"prompt_id": prompt_id},
+            )
+        else:
+            logger.warning(
+                "%s: failed to delete prompt (HTTP %s), continuing cleanup",
+                context,
+                exc.status_code,
+                extra={"prompt_id": prompt_id, "status_code": exc.status_code},
+            )
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)